{{cookiecutter.project_slug}}/docs/docker_ec2.rst

```diff
@@ -1,186 +0,0 @@
-    RUN ["groupadd", "python"]
-    RUN ["useradd", "python", "-s", "/bin/bash", "-m", "-g", "python", "-G", "python"]
-    ENV HOME /home/python
-    WORKDIR /home/python
-    RUN ["chown", "-R", "python:python", "/home/python"]
-    ADD ./ /home/python
-
-    # manage requirements
-    ENV REQUIREMENTS_REFRESHED_AT 2015-02-25
-    RUN ["pip3", "install", "-r", "requirements.txt"]
-
-    # uncomment the line below to use container as a non-root user
-    USER python:python
-
-Running `sudo docker-compose -f production.yml build` will follow the instructions in your `production.yml` file and build the database container, then your webapp, before mounting your cookiecutter project files as a volume in the webapp container and linking to the database. Our example yaml file runs in development mode but changing it to production mode is as simple as commenting out the line using `runserver` and uncommenting the line using `gunicorn`.
-
-Both are set to run on port `0.0.0.0:8000`, which is where the Docker daemon will discover it. You can now run `sudo docker-compose -f production.yml up` and browse to `localhost:8000` to see your application running.
-
-Deployment
-^^^^^^^^^^
-
-You'll need a webserver container for deployment. An example setup for `Nginx`_ might look like this:
-
-.. _Nginx: http://wiki.nginx.org/Main
-
-::
-
-    FROM ubuntu:14.04
-    ENV REFRESHED_AT 2015-02-11
-
-    # get the nginx package and set it up
-    RUN ["apt-get", "update"]
-    RUN ["apt-get", "-y", "install", "nginx"]
-
-    # forward request and error logs to docker log collector
-    RUN ln -sf /dev/stdout /var/log/nginx/access.log
-    RUN ln -sf /dev/stderr /var/log/nginx/error.log
-    VOLUME ["/var/cache/nginx"]
-    EXPOSE 80 443
-
-    # load nginx conf
-    ADD ./site.conf /etc/nginx/sites-available/your_cookiecutter_project
-    RUN ["ln", "-s", "/etc/nginx/sites-available/your_cookiecutter_project", "/etc/nginx/sites-enabled/your_cookiecutter_project"]
-    RUN ["rm", "-rf", "/etc/nginx/sites-available/default"]
-
-    #start the server
-    CMD ["nginx", "-g", "daemon off;"]
-
-That Dockerfile assumes you have an Nginx conf file named `site.conf` in the same directory as the webserver Dockerfile. A very basic example, which forwards traffic onto the development server or gunicorn for processing, would look like this:
-
-::
-
-    # see http://serverfault.com/questions/577370/how-can-i-use-environment-variables-in-nginx-conf#comment730384_577370
-    upstream localhost {
-        server webapp_1:8000;
-    }
-    server {
-        location / {
-            proxy_pass http://localhost;
-        }
-    }
-
-Running `sudo docker-compose -f production.yml build webserver` will build your server container. Running `sudo docker-compose -f production.yml up` will now expose your application directly on `localhost` (no need to specify the port number).
-
-Building and running your app on EC2
```

Comment:
I think this part is not described well enough in our current template docs, we should probably add an equivalent & updated section over there.